### PR TITLE
Minor fixes some rewording for clarity on Command Handlers

### PR DESCRIPTION
**Pull request by kipz**
One subtle but important change is making the difference between `commands` and `Command Handlers` clear. Commands are sent by the user and handled by Command Handlers. Commands are declared in Command Handlers. 

Relates to #105

### CODE REVIEW

**Comment by kipz** [comment]
Relates to #105

File: docs/user-guide/rug/commands.md [hunk 1]
```diff
@@ -20,16 +20,16 @@ or perform an action from where your team gathers to pilot the project.
 
 [bot]: /user-guide/interfaces/bot.md
 
-Depending on their goal, Rug commands are stored either along side the project
-they target or in a different project altogether when they are generic. In both
-cases however, Rug commands share the same approach as other Rugs. For the
-purpose of this documentation, we will assume our commands live in their own
-project rather than in an existing project like thie documentation describes for
-Rug generators or editors.
+Depending on their goal, Rug commands implementations are stored either along
```
Comment (ddgenome) [inline docs/user-guide/rug/commands.md]: "command implementations"

File: docs/user-guide/rug/commands.md [hunk 2]
```diff
@@ -188,44 +190,72 @@ export class FetchedFailed implements HandleResponse<any>{
...
+the Atomist bot, the Rug runtime runs the
+`#!typescript GetLatestSpringProjectTag` command handler.
+
+We then define the class which implements our command handler (line 11). A
+command handler implements the `#!typescript HandleCommand` interface. This
+interface requires the `#!typescript handle(command: HandlerContext): Plan`
+method to be defined (line 19). It is a convention for the command handler and
+the class that defines it to have the same name.
+
+Rug commands handlers can take parameters like other Rugs. These are declared
```
Comment (ddgenome) [inline docs/user-guide/rug/commands.md]: "command handlers"

File: docs/user-guide/rug/commands.md [hunk 3]
```diff
@@ -188,44 +190,72 @@ export class FetchedFailed implements HandleResponse<any>{
...
+caller, from the place where the user issued the command:
 
 ```typescript
 handle(response: Response<any>): Message {
     return new Message("Hello there!");
 }
 ```
 
-Next, the message is sent to a different place by setting a Slack channel identifier (we assume a Slack integration here):
+However, it is also possible to route to a different location. For by setting a
```
Comment (ddgenome) [inline docs/user-guide/rug/commands.md]: "different location by setting"

File: docs/user-guide/rug/commands.md [hunk 4]
```diff
@@ -235,11 +265,15 @@ handle(response: Response<any>): Message {
...
+A command handler can also return a `#!typescript Plan`. A plan describes the
+actions to be taken by the Rug runtime on behalf of the handler. Plans are
+composed of messages and/or instructions. The former are exactly as we describe
+above where instructions have the following properties:
 
 * `#!typescript kind`: the kind of instruction, one of `#!typescript "generate" | "edit" | "review" | "execute" | "respond" | "command"`
 * `#!typescript name`: the name of the operation to apply
-* `#!typescript parameters`: an object mapping properties to values and passed, by the Rug runtime, to the operation being called
+* `#!typescript parameters`: an object mapping names to values that is passed by
+the Rug runtime, to the Rug being called.
```
Comment (ddgenome) [inline docs/user-guide/rug/commands.md]: Does this render properly if it is not indented?

File: docs/user-guide/rug/commands.md [hunk 5]
```diff
@@ -254,20 +288,52 @@ To make this a little more concrete, our only instruction is defined as:
...
+`#!typescript Respond`. A `#!typescript Respond` indicates the desire to handle
+the response using a `#!typescript ResponseHandler`. A `#!typescript Plan` or
+`#!typescript Message` are handled by the Rug runtime just as if they were
+returned directly from the command handler as described above.
+
+Response handlers are declared with the `#!typescript ResponseHandler` decorator
+and the class must implement the
+`#!typescript handle(response: Response<T>): Plan` method of the
+`#!typescript HandleResponse<T>` interface. When the handler receives a JSON
+payload, you can benefit form an automatic deserialization into an object by
```
Comment (ddgenome) [inline docs/user-guide/rug/commands.md]: "benefit from"

**Comment by Lawouach** [comment]
LGTM

File: docs/user-guide/rug/commands.md (hunk 4 above)
Comment (kipz) [inline docs/user-guide/rug/commands.md]: Seems too, but will fix.

**Pull request merged**